<ui version="4.0" >
 <class>LocalServices</class>
 <widget class="QWidget" name="LocalServices" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="serviceList" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
